text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -248,6 +317,8 @@ def build_one_token_sequence_from_text(
         ----------
         text
             The raw text data of one sample.
+        is_training: 
+            Only augment for training data
```

Comment:
Need to explain what is `is_training` and then what it is used for.

Reply:
+1